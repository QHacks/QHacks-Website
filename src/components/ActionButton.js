import React from "react";

const ActionButton = (props) => <button css={props.style}>{props.text}</button>;

export default ActionButton;
