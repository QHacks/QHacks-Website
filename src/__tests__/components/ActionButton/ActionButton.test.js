import React from "react";
import ActionButton from "../../../components/ActionButton";
import { shallow } from "enzyme";

describe("ActionButton", () => {
  it("renders correctly", () => {
    expect(
      shallow(
        <ActionButton style={{ padding: "10px", margin: "10px" }} text="TEST" />
      )
    ).toMatchSnapshot();
  });
});
