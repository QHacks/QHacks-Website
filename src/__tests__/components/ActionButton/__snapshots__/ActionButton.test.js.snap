// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionButton renders correctly 1`] = `
<button
  css={
    Object {
      "margin": "10px",
      "padding": "10px",
    }
  }
>
  TEST
</button>
`;
